python/tvm/relay/frontend/tensorflow.py

```diff
@@ -793,6 +793,92 @@ def _impl(inputs, attr, params, mod):
     return _impl
 
 
+def convert_combined_nms_with_all_class_nms(
```

Comment:
Since I want to use this function to rewrite ONNX models, it is defined at the top level and visible from outside.